.editorconfig

```diff
@@ -14,5 +14,5 @@ trim_trailing_whitespace = false
 [*.php]
 indent_size = 4
 
-[resources/views/**.php]
```

Comment:
You can have regular `.php` files within resources/views. Those should be 2 spaces.

And blade files outside of resources/views should ... probably not be outside of resources/views, or they should be vendored.

Reply:
I'll push a change to handle `resources/views/**.php` – as far as Blade files outside of `resources/views` – there are multiple Sage-specific Gutenberg libraries popping up that are handling the views outside of `resources/views` such as:

- `app/Blocks/Example/Example.php`
- `app/Blocks/Example/views/example.blade.php`
- `app/Blocks/Example/views/example.(css,js)` 

which despite recommending separating things like this into an mu-plugin, is a somewhat clean approach to handling individual blocks without separating everything into 3 different directories when/if they are done inside of the theme.